google/cloud/spanner/integration_tests/client_integration_test.cc

```diff
@@ -356,10 +353,8 @@ void CheckReadWithOptions(
         options_generator) {
   using RowValues = std::vector<Value>;
   std::vector<RowValues> expected_rows;
-  auto commit = RunTransaction(
-      client, Transaction::ReadWriteOptions{},
-      [&expected_rows](Client const&,
-                       Transaction const&) -> StatusOr<Mutations> {
+  auto commit = client.Commit(
+      [&expected_rows](Transaction const&) -> StatusOr<Mutations> {
```

Comment:
FYI, all these call site examples in here really show how much nicer this new API is than the old one. Big improvement, IMO.